config/config.txt

```diff
@@ -21,6 +21,17 @@ HUBPASSWORD kMZy3U5jJHSiBQjr
 ## Add a # infront of this if you want to use the SQL based admin system, the legacy system uses admins.txt. You need to set up your database to use the SQL based system.
 ADMIN_LEGACY_SYSTEM
 
+## Add a # infront of this to disable the shutdown server verb.
+ALLOW_SHUTDOWN

```

Comment:
```suggestion
##ALLOW_SHUTDOWN
```
This should be disabled by default. It's more likely for downstreams to not be using TGS3.

Reply:
1 config file defaults are aimed at development defaults, not production defaults.

2 this does not need tgs to work, it just needs snowflake if you have tgs.

Reply:
Fair